src/Web/Sdk/Sdk.targets

```diff
@@ -19,6 +19,9 @@ Copyright (c) .NET Foundation. All rights reserved.
     <AddRazorSupportForMvc Condition="'$(DisableImplicitFrameworkReferences)' != 'true' And '$(AddRazorSupportForMvc)' != 'true'">true</AddRazorSupportForMvc>
   </PropertyGroup>
 
+  <!-- We can't add it here because the OutputPath and other msbuild properties are not evaluated.-->
+  <!--<Import Sdk="Microsoft.NET.Sdk.Publish" Project="ImportPublishProfile.targets" />-->
```

Comment:
@peterhuene - I tried using the default profile from here and ran into one issue. The default profile uses $(OutputPath) and this is set in the common.targets. So we would have to move the import logic to a place after the common.targets is done & before the NetSdk looks at the evaluated values.